core/network/address.go

```diff
@@ -212,16 +216,18 @@ func (a MachineAddress) ValueForCIDR(cidr string) (string, error) {
 // - machine-local next;
 // - link-local next;
 // - non-hostnames with unknown scope last.
+// Secondary addresses with otherwise equal weight will be sorted to come after
+// primary addresses, including host names *except* localhost.
 func (a MachineAddress) sortOrder() int {
 	order := 0xFF
```

Comment:
Any ideas why we use hex values here instead of plain ints?

Reply:
None. Might as well change it.